readme.md

```diff
@@ -1,39 +1,45 @@
-OWASP Nettacker
-=========
+<h1 align="center">
+  <br>
+  <a href="https://github.com/OWASP/Nettacker">
+  <img src="https://raw.githubusercontent.com/zdresearch/OWASP-Nettacker/master/web/static/img/owasp-nettacker.png" align="center" alt="SecureTea" width="100"></a>
+  <br>
+  OWASP Nettacker
+  <br>
+</h1>
+
+<h3 align="center">Made To Make Your Automated Pentesting Water🌊</h3>
```

Comment:
Sorry @MrinankBhowmick  while I appreciate your efforts in improving the format of README don't really understand why you think you can invent a "water" - related slogan for the project. It does not make sense and looks very un-professional.

Reply:
Very sorry sir @securestep9 , I should keep that in mind while adding this.

I just want to make that slogan 'short and catchy' so that more users attract to it and want to try this tool and contribute to it, eventually the project gets better and better.

Sir, you should change it to a better one before merging.

Thank you